homework/index.js

```diff
@@ -23,24 +23,147 @@
       const value = options[key];
       if (key === 'text') {
         elem.textContent = value;
+      } else if (key === 'html') {
+        elem.innerHTML = value;
       } else {
         elem.setAttribute(key, value);
       }
     });
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, repositories) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
+  const renderError = error => {
```

Comment:
Here you have chosen to use an (arrow) function expression assigned to a `const`. In other places there are regular function declarations, with the `function` keyword. It is best to be consistent and choose one style or the other.

My personal preference is to use regular function declarations for (named) functions at the file/global level and use (anonymous) arrow functions for callbacks and event handlers. This makes it easy to discriminate between data constants (defined with `const`) and functions (declared with `function`) at the file/global level.

Reply:
I changed all to regular function declaration.